linters/trufflehog/trufflehog.test.ts

```diff
@@ -1,2 +1,28 @@
-import { linterCheckTest } from "tests";
+import { customLinterCheckTest, linterCheckTest } from "tests";
+import { TrunkLintDriver } from "tests/driver";
+
 linterCheckTest({ linterName: "trufflehog" });
+
+// Create secrets that exist both only in history, and also in the working tree.
+const preCheck = async (driver: TrunkLintDriver) => {
+  if (driver.gitDriver) {
+    await driver.gitDriver.checkoutLocalBranch("branch-with-secrets");
+    driver.copyFileFromRootTo("linters/trufflehog/test_data/secrets.in.py", "secrets.in.py");
+    driver.copyFileFromRootTo("linters/trufflehog/test_data/secrets.in.py", "secrets2.in.py");
```

Comment:
At least the first call shouldn't be necessary? Everything in the test_data folder gets copied into the sandbox

Reply:
I think `customLinterCheckTest` doesn't copy those files? The `git add` fails without copying it myself first.

Reply:
```
    await driver.setUp();
    if (preCheck) {
      // preCheck is not always async, but we must await in case it is.
      await preCheck(driver);
      driver.debug("Finished running custom preCheck hook");
    }
```
preCheck should be called after any initial linter driver setup, so it should be there? Potentially worth investigating but priority is getting this landed